app/src/main/kotlin/com/wire/android/ui/home/conversations/model/messagetypes/multipart/standalone/EditableAssetPreview.kt

```diff
@@ -0,0 +1,174 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.android.ui.home.conversations.model.messagetypes.multipart.standalone
+
+import android.graphics.drawable.Drawable
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.style.TextOverflow
+import coil.compose.AsyncImage
+import coil.request.CachePolicy
+import coil.request.ImageRequest
+import com.wire.android.ui.common.applyIf
+import com.wire.android.ui.common.attachmentdraft.ui.FileHeaderView
+import com.wire.android.ui.common.colorsScheme
+import com.wire.android.ui.common.dimensions
+import com.wire.android.ui.common.multipart.MultipartAttachmentUi
+import com.wire.android.ui.common.progress.WireLinearProgressIndicator
+import com.wire.android.ui.home.conversations.messages.item.MessageStyle
+import com.wire.android.ui.home.conversations.messages.item.textColor
+import com.wire.android.ui.home.conversations.model.messagetypes.multipart.transferProgressColor
+import com.wire.android.ui.theme.Accent
+import com.wire.android.ui.theme.wireTypography
+import com.wire.kalium.logic.util.fileExtension
+
+/**
+ * Show preview for files which support online editing option.
+ */
+@Composable
+internal fun EditableAssetPreview(
+    item: MultipartAttachmentUi,
+    messageStyle: MessageStyle,
+    accent: Accent
+) {
+    Column(
+        modifier = Modifier
+            .applyIf(messageStyle == MessageStyle.BUBBLE_SELF) {
+                background(
+                    colorsScheme().bubbleContainerAccentBackgroundColor.getOrDefault(
+                        accent,
+                        colorsScheme().defaultBubbleContainerBackgroundColor
+                    )
+                )
```

Comment:
Please use `background(colorsScheme().selfBubble.secondary))`
I just noticed that these XAssetPreview composables haven't been updated after implementing updates to the bubble designs and I'm raising a PR with that fix, but basically `bubbleContainerAccentBackgroundColor` and `defaultBubbleContainerBackgroundColor` should be already removed in favor of `selfBubble.secondary` and `otherBubble.secondary`